app/javascript/src/modules/embed_this.js

```diff
@@ -1,42 +1,39 @@
 // Module handles Embed This form interaction
 
-(function( global ) {
-  var Module = (function() {
-    var formContainer;
-    return {
-      init: function() {
-        var _this = this;
-        $(".sul-embed-embed-this-form").each(function(){
-          formContainer = $(this);
-          _this.observeEmbedForm();
-        });
-      },
-      observeEmbedForm: function() {
-        var _this    = this,
-            textarea = $('textarea', formContainer);
-        $('input[type="checkbox"], input[type="text"]', formContainer).on('change', function(){
-          var checked   = $(this).is(':checked'),
-              inputType = $(this).attr('type'),
-              src       = textarea.text().match(/src="(\S+)"/)[1],
-              urlAttr   = '&' + $(this).data('embed-attr') + '=true';
-          if(inputType === 'checkbox'){
-            if(checked) {
-              textarea.text(textarea.text().replace(urlAttr, ''));
-            }else{
-              textarea.text(textarea.text().replace(src, src + urlAttr));
-            }
+const EmbedThis = (function() {
+  let formContainer;
+  return {
```

Comment:
This is mostly an indent change. It may be easier to review if you select "Hide whitespace"